### PR TITLE
[VEN-2875] [VEN-2877] Deploy XVS and XVS Bridge on Base Sepolia and Mainnet

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

**Review by chechu — COMMENTED** [review]
Testnet commands lgtm

**Comment by chechu** [comment]
Let's keep this PR open until adding the mainnet commands

File: hardhat.config.zksync.ts [hunk 1]
```diff
@@ -162,7 +162,7 @@ const config: HardhatUserConfig = {
     },
     zksynctestnode: {
       url: process.env.ZKSYNC_ERA_LOCAL_TEST_NODE || "http://localhost:8011",
-      chainId: 300, // change it to 324 for zksyncmainnet
+      chainId: 324, // change it to 324 for zksyncmainnet
```
Comment (GitGuru7) [inline hardhat.config.zksync.ts:165]:
```suggestion
      chainId: 300, // change it to 324 for zksyncmainnet
```

File: src/types.ts [hunk 2]
```diff
@@ -14,9 +14,15 @@ export type SUPPORTED_NETWORKS =
...
-export const REMOTE_TESTNET_NETWORKS = ["sepolia", "opbnbtestnet", "arbitrumsepolia", "zksyncsepolia", "opsepolia"];
+export const REMOTE_TESTNET_NETWORKS = [
+  "sepolia",
+  "opbnbtestnet",
+  "arbitrumsepolia",
+  "zksyncsepolia",
+  "opsepolia",
+  "basesepolia",
+];
 export const REMOTE_MAINNET_NETWORKS = ["ethereum", "opbnbmainnet", "arbitrumone", "zksyncmainnet", "opmainnet"];
```
Comment (GitGuru7) [inline src/types.ts:25]:
```suggestion
export const REMOTE_MAINNET_NETWORKS = ["ethereum", "opbnbmainnet", "arbitrumone", "zksyncmainnet", "opmainnet", "basemainnet"];
```

File: src/utils.ts [hunk 3]
```diff
@@ -394,7 +394,11 @@ export const setMaxStalePeriod = async (
     FORKED_NETWORK == "ethereum" ||
     FORKED_NETWORK == "sepolia" ||
     FORKED_NETWORK == "opbnbmainnet" ||
-    FORKED_NETWORK == "opbnbtestnet"
+    FORKED_NETWORK == "opbnbtestnet" ||
+    FORKED_NETWORK == "opmainnet" ||
+    FORKED_NETWORK == "opsepolia" ||
+    FORKED_NETWORK == "zksyncmainnet" ||
+    FORKED_NETWORK == "zksyncsepolia"
```
Comment (GitGuru7) [inline src/utils.ts:401]: We should add base network here as well, IMO.

File: type-extensions.ts [hunk 4]
```diff
@@ -6,6 +6,7 @@ declare module "hardhat/types/runtime" {
     FORKED_NETWORK?:
       | "bsctestnet"
       | "bscmainnet"
+      | "basesepolia"
```
Comment (GitGuru7) [inline type-extensions.ts:9]:
```suggestion
```

File: hardhat.config.zksync.ts (hunk 1 above)
Comment (chechu) [reply hardhat.config.zksync.ts]: https://github.com/VenusProtocol/vips/pull/424/commits/ac889b152cd2358304c20c252882b66a8429c66f

File: src/types.ts (hunk 2 above)
Comment (chechu) [reply src/types.ts]: https://github.com/VenusProtocol/vips/pull/424/commits/ac889b152cd2358304c20c252882b66a8429c66f

File: src/utils.ts (hunk 3 above)
Comment (chechu) [reply src/utils.ts]: https://github.com/VenusProtocol/vips/pull/424/commits/ac889b152cd2358304c20c252882b66a8429c66f

File: type-extensions.ts (hunk 4 above)
Comment (chechu) [reply type-extensions.ts]: https://github.com/VenusProtocol/vips/pull/424/commits/ac889b152cd2358304c20c252882b66a8429c66f

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/407?chainId=56

**Pull request merged**